Add CHANGELOG for v0.10 (#38)
turbot/steampipe-mod-aws-tags: CHANGELOG.md

## v0.10 [2023-08-08]

_Bug fixes_

> - Fixed the query used in the controls of the `untagged` benchmark to correctly check for resources where tags are `{}`. ([#37](https://github.com/turbot/steampipe-mod-aws-tags/pull/37)) (Thanks [@brad-webb](https://github.com/brad-webb) for the contribution!!)

Reviewer: ```suggestion
- Fixed the `untagged` benchmark to correctly check if resources have tags or not. ([#37](https://github.com/turbot/steampipe-mod-aws-tags/pull/37)) (Thanks [@brad-webb](https://github.com/brad-webb) for the contribution!!)
```

> - Fixed the query used in the controls of the `expected_tag_values` benchmark to skip those resources where either tags is not set or there are no matching tag keys. ([#33](https://github.com/turbot/steampipe-mod-aws-tags/pull/33))

Reviewer: ```suggestion
- Fixed the `expected_tag_values` benchmark to skip resources where either tags are not set or there are no matching tag keys. ([#33](https://github.com/turbot/steampipe-mod-aws-tags/pull/33))
```
